jobs/build-webapp.groovy

```diff
@@ -460,6 +460,11 @@ def deployToGatewayConfig() {
        exec(["make", "-C", "services/graphql-gateway",
```

Comment:
Should we add a TODO here to remove when JS is no longer used?

Reply:
The expectation is when we get rid of the js we'll rename the go gateway to be services/graphql-gateway, and get rid of services/graphql-gateway-2.  That's why I have the `if` check as well.  (I talk about this a bit in the commit message.)

Reply:
> The expectation is when we get rid of the js we'll rename the go gateway to be services/graphql-gateway, and get rid of services/graphql-gateway-2. That's why I have the `if` check as well. (I talk about this a bit in the commit message.)

Awesome! Yup - might be worth adding a note around that expectation.